botocore/__init__.py

```diff
@@ -51,6 +51,9 @@ def emit(self, record):
 
 BOTOCORE_ROOT = os.path.dirname(os.path.abspath(__file__))
 
+# Used to specify anonymous (unsigned) request signature
+UNSIGNED = ''
```

Comment:
Is the auth override stuff even in a public release yet?  If the only reason for this is that it's not backwards compatible, and not that there's an issue with using `object()` sentinel's, I'd vote for changing this to be consistent.  Pretty sure the impact here is minimal, if any.